### DISCUSSION TITLE
黄奇帆的观点

### DISCUSSION THREAD

**Discussion by portwatcher** [portwatcher/blog-comments#47] [Announcements]
https://juryquinn.com/post/books/2021-10-24

**Comment by imxana** [comment]
我大概了解了下经济学的知识，大概了解了房价的价格就等于这个城市居民中位数收入+贷款额度，举个不恰当的例子，你能极限贷款个50w，那房价就是50w，你能贷个500w，那房价就是500w。这和合理也并没有什么关系。我国经济发展基本上就是高增长+低负债的结果，所以几十年来房价上涨其实也符合客观逻辑，高增长+低负债，房价的极限就是你007的所有收入，不可能超过这个极限。

**Comment by imxana** [comment]
所以纯谈收入讲房价并不合适 [英国留学生平均负债35万，需88年才能还清](https://zhuanlan.zhihu.com/p/83323545) 

房价就是设计成一半人买不起的价格的，tg为了维稳，否则早就gm了。[开发商降价被“约谈”](https://www.163.com/dy/article/E0LD95940523WJSP.html)

**Comment by imxana** [comment]
关于供需部分：

那为啥房价敢这么定呢，因为他是生活必需品而且稀缺，所以是被垄断而制造的人为稀缺，而生活必需品被垄断就要会表现出这样的特质：把你的未来透支完为止，996榨不干你，007总榨干了吧。

我们缺房子吗，不缺，但是为什么这么贵呢，因为被垄断。而房子被谁垄断呢，你知道我也知道，但是为什么这个人不说呢，因为他是党员

**Comment by imxana** [comment]
关于货币因素：

黄奇帆明显是吧房市当做了蓄水池，但是房市除了部分情况是蓄水池，很明显有放水的作用，房价一跌，水就少了。[第二季度的三道红线](https://baijiahao.baidu.com/s?id=1703974967684191610&wfr=spider&for=pc)同时期的[M2指数环比下跌](http://data.eastmoney.com/cjsj/hbgyl.html)

普通消费品逻辑和生活必需品这个是初中知识了，因为它不稀缺，所以售价只和供需有关，购买力下降，供给过剩，怎么可能物价上涨。[中国 工业品出厂价格指数(PPI)直线上升](https://data.eastmoney.com/cjsj/ppi.html)同时期的[中国 居民消费价格指数(CPI)](https://data.eastmoney.com/cjsj/cpi.html)下跌，根本没发生通胀。只能说老党员有水平

**Comment by imxana** [comment]
所以打工人是最爱国的，自愿帮助国家搞垄断td财政，响应国家艰苦奋斗，这样国家就有钱搞基建，下贷地方zf，人打工在国内，消费也在国内。

金融资本是没有祖国的，必须打死

**Comment by imxana** [comment]
（完）
